doc/api/timers.md

```diff
@@ -532,9 +532,8 @@ added:
 An experimental API defined by the [Scheduling APIs][] draft specification
 being developed as a standard Web Platform API.
 
-Calling `timersPromises.scheduler.wait(delay, options)` is roughly equivalent
-to calling `timersPromises.setTimeout(delay, undefined, options)` except that
-the `ref` option is not supported.
+Calling `timersPromises.scheduler.wait(delay, options)` is equivalent
+to calling `timersPromises.setTimeout(delay, undefined, options)`.
```

Comment:
Because the documentation was incorrect. It was not `roughly` equivalent, it was `strictly` equivalent since it just wrapping it. I made that explicit.

Reply:
Then I guess we should document `ref` as a valid option